spec/models/vote_spec.rb

```diff
@@ -23,4 +23,42 @@
       expect(place2_vote.valid?).to be_truthy
     end
   end
+
+  context "most and least scenic places" do
+    let(:most_scenic_place) { FactoryBot.create(:place) }
+    let(:least_scenic_place) { FactoryBot.create(:place) }
+
+    before do
+      FactoryBot.create_list(:vote, 3, place: most_scenic_place, rating: 10)
+      FactoryBot.create_list(:vote, 3, place: least_scenic_place, rating: 1)
+      FactoryBot.create_list(:vote, 5, rating: 8)
+      FactoryBot.create_list(:vote, 5, rating: 3)
+    end
+
+    describe ".most_scenic_places" do
+      it "returns the top 5 rated places" do
+        top_five = Vote.most_scenic_places
+
+        expect(top_five.size).to eql(5)
+        expect(top_five.first).to match(hash_including(
+          "place_id" => most_scenic_place.id,
+          "score" => 10,
+          "vote_count" => 3
+        ))
+      end
```

Comment:
The thinking is that there will be enough data in production (since we'll be migrating all the ratings) to produce viable results.

I've already modified the original logic a bit to be more permissive (since I didn't have enough data in development, and it felt more useful than generating more seed data).

We'll also make sure we discuss the logic for determining the top fives, since the original reasoning might be lost to history, or might not hold up to the presumed future influx of fresh votes.